CONTRIBUTORS.md

```diff
@@ -0,0 +1,44 @@
+This is a list of contributors to the OpenTimelineIO project, sorted alphabetically by first name.
+
+If you know of anyone missing from this list, please contact us: https://lists.aswf.io/g/otio-discussion
+
+* ? ([bashesenaxis](https://github.com/bashesenaxis))
+* ? ([elabrosseRodeofx](https://github.com/elabrosseRodeofx))
```

Comment:
@elabrosseRodeofx would you like your full name listed here?

Reply:
Yes please. I'd prefer 'Emile Labrosse' If possible.
Thanks !